yt/visualization/plot_window.py

```diff
@@ -1079,6 +1081,9 @@ def hide_axes(self, field=None):
         field : string, field tuple, or list of strings or field tuples (optional)
             The name of the field(s) that we want to hide the axes.
 
+        draw_frame : boolean
```

Comment:
I'm not sure to understand what `draw_frame` is actually doing? From the parameter description, I understand that `hide_axes` still draws the axes if `draw_frame==True`?
Maybe adding a note down there would be worse it, something like
```
draw_frame:boolean
   If True, the axes frame will still be drawn. Defaults to False. See note below for more details
[…]

Note
------
By default, when removing the axes, the frame on which the axes are drawn is not displayed, making it impossible to later change e.g. the background colour. It is possible to force the frame to be displayed using the `draw_frame` argument.```